inlong-sort/sort-end-to-end-tests/sort-end-to-end-tests-v1.15/src/test/java/org/apache/inlong/sort/tests/Mongodb2StarRocksTest.java

```diff
@@ -131,7 +103,7 @@ public void setup() {
         initializeStarRocksTable();
     }
 
-    private void initializeStarRocksTable() {
+    public static void initializeStarRocksTable() {
```

Comment:
use initializeStarRocksTable in StarRocksManager and delete initializeStarRocksTable in Mongodb2StarRocksTest

Reply:
I try it, but  when starrocks container insert named "id" column will account exception. So, in Mongodb2StarRocksTest, the column is "_id".

Reply:
Maybe you can pass the primary key to initializeStarRocksTable ?

Reply:
fix it, thx~